lib/src/migrator.dart

```diff
@@ -45,7 +48,12 @@ abstract class Migrator extends Command<Map<Uri, String>> {
   Map<Uri, String> run() {
     var allMigrated = Map<Uri, String>();
     for (var entrypoint in argResults.rest) {
-      var migrated = migrateFile(canonicalize(Uri.parse(entrypoint)));
+      var canonicalUrl = canonicalize(Uri.parse(entrypoint));
+      if (canonicalUrl == null) {
+        throw MigrationException(
+            "Error: Could not find Sass file at '$entrypoint'.");
```

Comment:
Should entrypoint go through `p.prettyUri()`?

Reply:
Since `entrypoint` comes directly from the arguments passed to the executable, I figured it made more sense to just error with the exact text entered by the user.

Reply:
SGTM